<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>96</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="SerialGroupBox">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>275</height>
      </size>
     </property>
     <property name="title">
      <string>Serial Port</string>
     </property>
     <widget class="QGroupBox" name="additionalOptionsGroupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>205</y>
        <width>350</width>
        <height>60</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>60</height>
       </size>
      </property>
      <property name="title">
       <string>Additional options</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="localEchoCheckBox">
         <property name="text">
          <string>Local echo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="parametersBox">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>160</width>
        <height>180</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>180</height>
       </size>
      </property>
      <property name="title">
       <string>Select Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="baudRateLabel">
         <property name="text">
          <string>BaudRate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="baudRateBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dataBitsLabel">
         <property name="text">
          <string>Data bits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="dataBitsBox"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="parityLabel">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="parityBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="stopBitsLabel">
         <property name="text">
          <string>Stop bits:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="stopBitsBox"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="flowControlLabel">
         <property name="text">
          <string>Flow control:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="flowControlBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="selectBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>180</width>
        <height>180</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>180</height>
       </size>
      </property>
      <property name="title">
       <string>Select Serial Port</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QComboBox" name="serialPortInfoListBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="descriptionLabel">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="manufacturerLabel">
         <property name="text">
          <string>Manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="serialNumberLabel">
         <property name="text">
          <string>Serial number:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="locationLabel">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="vidLabel">
         <property name="text">
          <string>Vendor ID:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="pidLabel">
         <property name="text">
          <string>Product ID:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QGroupBox" name="ModGroupBox">
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>285</height>
      </size>
     </property>
     <property name="title">
      <string>ModBus</string>
     </property>
     <widget class="QGroupBox" name="additionalOptionsGroupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>197</y>
        <width>350</width>
        <height>80</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>80</height>
       </size>
      </property>
      <property name="title">
       <string>Additional options</string>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="rtLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>response timeout:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="norLabel">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>20</height>
          </size>
         </property>
         <property name="text">
          <string>number of retrys:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="rtLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="norLineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="parametersBox_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>160</width>
        <height>170</height>
       </rect>
      </property>
      <property name="title">
       <string>Select Parameters</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLabel" name="baudRateLabel_2">
         <property name="text">
          <string>BaudRate:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="baudRateBox_2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="dataBitsLabel_2">
         <property name="text">
          <string>Data bits:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="parityLabel_2">
         <property name="text">
          <string>Parity:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="stopBitsBox_2"/>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="parityBox_2"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="stopBitsLabel_2">
         <property name="text">
          <string>Stop bits:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="dataBitsBox_2"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="selectBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>180</width>
        <height>170</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>170</height>
       </size>
      </property>
      <property name="title">
       <string>Select ModBus Port</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QComboBox" name="serialPortInfoListBox_2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>75</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="descriptionLabel_2">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="manufacturerLabel_2">
         <property name="text">
          <string>Manufacturer:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="serialNumberLabel_2">
         <property name="text">
          <string>Serial number:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="locationLabel_2">
         <property name="text">
          <string>Location:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="vidLabel_2">
         <property name="text">
          <string>Vendor ID:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="pidLabel_2">
         <property name="text">
          <string>Product ID:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
